### PR TITLE
Address early feedback from OpenPGP mailing list

### PR DESCRIPTION
**Pull request by milesmcc**
**This is a draft pull request.** It addresses much of the feedback from Paul and Wiktor (@wiktor-k) on the OpenPGP mailing list.

Note that these changes are still a bit rough around the edges, and could use some polish before we publish them in v04.

### CODE REVIEW

File: rfc.xml [hunk 1]
```diff
@@ -435,34 +450,55 @@
...
-					The keylist subscription functionality defined in this
-					document provide a number of security benefits, including:
+					Unlike Web Key Directories, keylists are not domain specific. A keylist
+					might contain public key fingerprints for email addresses across several
+					different domains. Moreover, keylists only provide references to public
+					keys by way of fingerprints; Web Key Directories provide the public keys
+					themselves.
 				</t>
+			</section>
+			<section anchor="dns" title="OPENPGPKEY DNS Records">
```
Comment (wiktor-k) [inline rfc.xml]: I'll mention one interesting thing that may be relevant (or not). Another I-D - [security.txt uses URIs to refer to PGP keys](https://tools.ietf.org/html/draft-foudil-securitytxt-05#section-3.5.4). That gives them ability to cover in one consistent way a) OPENPGPKEY DNS based keys, b) V4 fingerprints and even c) HTTPS links to keys.

**Comment by wiktor-k** [comment]
I've read the changes. They look good :+1:

File: rfc.xml [hunk 2]
```diff
@@ -92,17 +93,17 @@
...
-				cryptographic verification, and the method by which it is
-				retreived by the client. The user interface by which a client
-				provides this functionality to the user is out of scope, as is
-				the process by which the client retrieves public keys. Other
-				non-security-related implementation details are also out of
-				scope.
+				 This document specifies a system by which clients may subscribe to
+				 cryptographically signed 'keylists' of public key fingerprints. The public
+				 keys do not necesssarily all correspond to a single domain. This system
+				 enhances operational security by allowomg seamless key rotation across
```
Comment (micahflee) [inline rfc.xml]: allowing*

File: rfc.xml [hunk 3]
```diff
@@ -435,34 +450,55 @@
...
+					 outside of the control of an organization's system adminitrators.
 				</t>
 			</section>
-			<section anchor="securitydrawbacks" title="Security Drawbacks">
+			<!-- We'll want to clean this up, but the messaging we'd like to convey is there. -->
+		</section>
+		<section anchor="securityconsiderations" title="Security Considerations">
 				<t>
 					There is a situation in which keylist subscriptions could
 					pose a potential security threat. If the both the authority
```
Comment (micahflee) [inline rfc.xml]: Found an old typo. "If the both the" should be "If both the"

**Comment by micahflee** [comment]
Same 👍

File: rfc.xml (hunk 1 above)
Comment (milesmcc) [reply rfc.xml]: Hmm, that's interesting. Might be something we include in v05? @micahflee would allowing URIs seriously complicate things in the implementation? If not, we might consider including it. It'd resolve a lot of the ambiguity around fingerprints, which kinds to allow, and give more flexibility in key sourcing.

I'll create an issue for this.

Comment (milesmcc) [reply rfc.xml]: See #12

File: rfc.xml (hunk 2 above)
Comment (milesmcc) [reply rfc.xml]: Good catch. Fixed.

File: rfc.xml (hunk 3 above)
Comment (milesmcc) [reply rfc.xml]: Fixed!

**Pull request merged**